src/ProblemDetailsResponseFactory.php

```diff
@@ -330,15 +334,15 @@ private function cleanKeysForXml(array $input): array
     {
         $return = [];
         foreach ($input as $key => $value) {
-            $key = str_replace("\n", '_', $key);
+            $key                   = str_replace("\n", '_', (string) $key);
```

Comment:
In PHP 8, a TypeError is thrown if the third param for `str_replace` is not a string or null.

One of the tests was failing because the keys were ints, and this was the attempt to fix it. It works, but the implicit casting could hide other problems, so I'm open for suggestions.

Reply:
Cast seems correct 👍